packages/hoppscotch-sh-admin/locales/en.json

```diff
@@ -156,11 +156,11 @@
     "self_host_docs": "Self Host Documentation",
     "send_magic_link": "Send magic link",
     "setup_failure": "Setup has failed!!",
-    "setup_success": "Setup completed successful!!",
+    "setup_success": "Setup completed successfully!!",
     "sign_in_agreement": "By signing in, you are agreeing to our",
     "sign_in_options": "All sign in option",
     "sign_out": "Sign out",
-    "team_name_long": "Workspace name should be atleast 6 characters long!!",
+    "team_name_too_short": "Workspace name should be atleast 6 characters long!!",
```

Comment:
The locale entry wasn't updated for the following instances.

https://github.com/hoppscotch/hoppscotch/blob/4798d7bbbd895cef1736eb9b586d9005bc15f71b/packages/hoppscotch-sh-admin/src/components/teams/Details.vue#L138-L141

https://github.com/hoppscotch/hoppscotch/blob/4798d7bbbd895cef1736eb9b586d9005bc15f71b/packages/hoppscotch-sh-admin/src/pages/teams/index.vue#L182-L185